analysis/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json"
```

Comment:
does this work if `gts` is hoisted differently ? It should probably use `gts/tsconfig-google.json` instead, relying on the node-style resolution of the package